sdk/src/Services/S3/Custom/Transfer/_async/ITransferUtility.async.cs

```diff
@@ -237,6 +237,37 @@ public partial interface ITransferUtility : IDisposable
         /// <returns>The task object representing the asynchronous operation.</returns>
         Task<Stream> OpenStreamAsync(TransferUtilityOpenStreamRequest request, CancellationToken cancellationToken = default(CancellationToken));
 
+        /// <summary>
```

Comment:
Since this is the main API public documentation this should be expanded with details about the buffering and where to adjust things. With this description I wouldn't see this method any different then existing other then I get a response.